public/.htaccess

```diff
@@ -0,0 +1,21 @@
+<IfModule mod_rewrite.c>
```

Comment:
Not related to this file really but are we running on apache or nginx ?

Reply:
![image](https://user-images.githubusercontent.com/21128579/93008239-7f428780-f540-11ea-99da-cb4e1c8f530d.png)
Answer ^ :eyes:

Reply:
Should be nginx right @alexstojda ?

Reply:
Will switch to nginx

Reply:
Actually will open separate ticket for that, needs a bit of extra work to setup in the dev container #5